x/oracle/client/cli/txUpgradeOracleProposal.go

```diff
@@ -0,0 +1,89 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	FlagUpgradeUniqueID = "upgrade-unique-id"
+	FlagUpgradeHeight   = "upgrade-height"
+)
+
+func CmdUpgradeOracleProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "oracle-upgrade (--upgrade-unique-id [uniqueID]) (--upgrade-height [height]) [flags]",
+		Args:  cobra.ExactArgs(0),
+		Short: "Submit a oracle upgrade proposal",
+		Long: "Submit a oracle upgrade along with an initial deposit.\n + " +
+			"You must enter the uniqueID and block height of the Oracle to perform the upgrade.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			from := clientCtx.GetFromAddress()
+
+			depositStr, err := cmd.Flags().GetString(govcli.FlagDeposit)
+			if err != nil {
+				return err
+			}
+
+			deposit, err := sdk.ParseCoinsNormalized(depositStr)
+			if err != nil {
+				return err
+			}
+
+			content, err := parseArgsToContent(cmd)
+			if err != nil {
+				return err
+			}
+
+			msg, err := govtypes.NewMsgSubmitProposal(content, deposit, from)
+			if err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().String(govcli.FlagTitle, "", "title of proposal")
+	cmd.Flags().String(govcli.FlagDescription, "", "description of proposal")
+	cmd.Flags().String(govcli.FlagDeposit, "", "deposit of proposal")
+	cmd.Flags().Int64(FlagUpgradeHeight, 0, "The height at which the upgrade must happen")
+	cmd.Flags().String(FlagUpgradeUniqueID, "", "Oracle's uniqueID to be upgraded")
+
+	return cmd
+}
+
+func parseArgsToContent(cmd *cobra.Command) (govtypes.Content, error) {
+	title, err := cmd.Flags().GetString(govcli.FlagTitle)
+	if err != nil {
+		return nil, err
+	}
+
+	description, err := cmd.Flags().GetString(govcli.FlagDescription)
+	if err != nil {
+		return nil, err
+	}
+
+	height, err := cmd.Flags().GetInt64(FlagUpgradeHeight)
+	if err != nil {
+		return nil, err
+	}
+
+	uniqueID, err := cmd.Flags().GetString(FlagUpgradeUniqueID)
+	if err != nil {
+		return nil, err
+	}
+
+	plan := types.Plan{UniqueId: uniqueID, Height: height}
```

Comment:
We can check the validity of plan from cmd using `plan.ValidateBasic()` before generate tx

Reply:
Good. I'll add